ActindoWorkFlow/Integration/01_ConfigureQueueTypes.md

```diff
@@ -273,19 +282,38 @@ Kill all workers to stop all currently running actions.
 
 #### Prerequisites
 
-At least one queue type has been created, see [Create a queue type](#create-a-queue-type).
+The workflows are disabled. Disable the workflows before you kill the workers to prevent an automatic restart of the workflow processes, see [Disable the workflows](#disable-the-workflows).
 
-> [Info] The *Default* queue type is predefined.
 
 #### Procedure
 
-*Workflows > Queue types > Tab OVERVIEW*
+*Process Orchestration > Queue types > Tab OVERVIEW*
 
 ![Overview](../../Assets/Screenshots/ActindoWorkFlow/QueueTypes/Overview.png "[Overview]")
 
-> [Info] Disable the workflows before you kill the workers to prevent an automatic restart of the workflow processes, see [Disable the workflows](#disable-the-workflows).
+1. Click the [KILL WORKER] button above the queue types list.    
+    All workers have been killed. The process actions will not continue until you restart the workflow processes. A confirmation message is displayed.
+
+2. Unfold the engine room panel, disable the *Disable workflow* toggle in the *Actindo Work Flow Engine* box and fold the engine room panel again.
+    The workflow processes will start automatically after one minute.
+
+
+
+## Start workers
+
+Sometimes it may be necessary to start the process actions manually. For example, if you have previously disabled the *Disable workflow* toggle in the *Actindo Work Flow Engine* box of the engine room panel, and you want the workers to start immediately.
```

Comment:
... if you have previously disabled the *Disable workflow* toggle ... **and killed the workers...**?

Reply:
done